compiler/plugins/explain.ml

```diff
@@ -1361,11 +1433,20 @@ let to_dot lang ppf ctx env base_vars g ~base_src_url ~line_format =
        *     out_funs with
        *     Format.out_newline = (fun () -> out_funs.out_string "<br/>" 0 2);
        *   }; *)
-      expr_to_dot_label env ctx lang ppf e
+      expr_to_dot_label theme env ctx lang ppf e
     (* ; * Format.pp_print_flush ppf (); * Format.pp_set_formatter_out_functions
        ppf out_funs *)
 
-    let graph_attributes _ = [ (* `Rankdir `LeftToRight *) ]
+    let graph_attributes _ =
+      [
+        `BgcolorWithTransparency (Int32.of_int 0x00);
+        (* `Ratio (`Float 0.8); *)
+        (* `Concentrate true; *)
```

Comment:
Should these be toggable with an option?

Reply:
Hm I was playing around trying to improve the layout, but from my tests they don't really provide meaningful improvements ; I certainly should cleanup all these comments though.